src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +39,7 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" onClick={this.handleBackButtonClick} to="/">Retour</Link>
```

Comment:
Interpolate the URL directly into the `href` attribute so that the browser shows the destination URL.